### PR TITLE
Multiple fixes

### PR DESCRIPTION
**Pull request by xlc**
Fix estimate gas, gas limit in block, docker compile file, get transaction receipt

closes https://github.com/AcalaNetwork/bodhi.js/issues/97

### CODE REVIEW

File: .vscode/extensions.json [hunk 1]
```diff
@@ -0,0 +1,3 @@
+{
+  "recommendations": ["mrmlnc.vscode-json5"]
```
Comment (shunjizhan) [inline .vscode/extensions.json:2]: quick question: should we commit VScode setting files or keep them local?

Comment (xlc) [reply .vscode/extensions.json:2]: this is recommended extension and intended to be committed so others know they should install this.

**Review by shunjizhan — APPROVED** [review]

File: evm-subql/docker-compose.yml [hunk 2]
```diff
@@ -29,44 +29,44 @@ services:
...
       DB_DATABASE: postgres
-      DB_HOST: localhost
+      DB_HOST: postgres
       DB_PORT: 5432
       PORT: 3001
     command:
       - --name=acala-evm
       - --playground
-      - --indexer=http://localhost:3000
+      - --indexer=http://subquery-node:3000
```
Comment (gluneau) [inline evm-subql/docker-compose.yml:69]: Does this mean that I have to add these aliases to my `hosts` for running a local testnet?

`127.0.0.1       localhost subquery-node postgres`

Comment (xlc) [reply evm-subql/docker-compose.yml:69]: no. docker have its internal dns resolver does which will be used by containers

**Pull request merged**